### PR TITLE
Adding detection for new adware variants to osx-attacks

### PR DESCRIPTION
**Pull request by Centurion89**
• Added additional checks for Vsearch
• Added detection for PremiereOpinion, Bundlore, Spigot, and SearchInstUpdater

### CODE REVIEW

**Comment by osqueryer** [comment]
Can one of the osquery admins reply with "ok to test" to kick off a build...

**Comment by theopolis** [comment]
ok to test

**Comment by osqueryer** [comment]
Build origin/pr/1970/merge (Job: 2521) is good! :+1:

File: packs/osx-attacks.conf [hunk 1]
```diff
@@ -206,6 +206,30 @@
...
+      "value": "Artifact used by this malware"
+    },
+    "Spigot": {
+      "query": "select * from launchd where name = 'com.spigot.ApplicationManager.plist' or name = 'com.spigot.SearchProtection.plist';",
+      "interval": "86400",
+      "description": "(http://www.thesafemac.com/arg-spigot/)",
+      "value": "Artifact used by this malware"
+    },
+    "SearchInstUpdater": {
+      "query": "select * from launchd where name = 'com.updater.mce.plist' or name = 'com.updater.watch.mce.plist' or name = 'com.updater.mcy.plist' OR name = 'com.updater.watch.mcy.plist';",
```
Comment (theopolis) [inline packs/osx-attacks.conf]: Could this set of launch daemon names be minimized with some wildcards?

Comment (Centurion89) [inline packs/osx-attacks.conf]: Yep, totally. Updated in a new commit.

**Comment by osqueryer** [comment]
Build origin/pr/1970/merge (Job: 2523) is good! :+1:

**Comment by theopolis** [comment]
Perfect, thanks @Centurion89!

**Pull request merged**